jackal_base/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ find_package(catkin REQUIRED COMPONENTS
 find_package(Boost REQUIRED COMPONENTS thread)
 
 catkin_package()
+catkin_add_env_hooks(50.jackal_find_mag_config
```

Comment:
I didn't know about this. For others who are reviewing: http://docs.ros.org/indigo/api/catkin/html/dev_guide/generated_cmake_api.html#catkin-add-env-hooks